nodejs/index.js

```diff
@@ -23,14 +23,20 @@ function getHandler() {
 
   const parts = handler.split('.')
 
-  if (parts.length !== 2) {
+  let moduleToImport
+  let handlerToWrap
+
+  if (parts.length === 2) {
```

Comment:
can't you just keep the new lastIndexOf logic for all?

You could do something like this

```
const lastDotIndex = handler.lastIndexOf('.')
const moduleToImport = handler.slice(0, lastDotIndex)
const handlerToWrap = handler.slice(lastDotIndex + 1)
```

Reply:
@bizob2828 True--that would indeed apply to any length > 1. 

I did want to hold off on linking the issue until this was in full PR--I want to test first with a locally-built layer and an application with some dot directories.

Reply:
got it.  was there a reason why this doesn't have tests? It would be really easy to unit test this

Reply:
The lack of tests came down to team size vs. the size of the docket of work. Not so much an explicit decision not to use tests, but more a lack of a decision to use them.